src/js/form-model.js

```diff
@@ -584,20 +505,8 @@ FormModel.prototype.mergeXml = function (recordStr) {
 
     // Remove the primary instance childnode from the original model
     this.xml.querySelector('instance').removeChild(modelInstanceChildEl);
```

Comment:
No need to change now, noting for posterity: I think by using `adoptNode`, this can be removed.